code/modules/mob/living/living_mobility.dm

```diff
@@ -96,7 +96,14 @@
 		mobility_flags &= ~MOBILITY_STAND
 		setMovetype(movement_type | CRAWLING)
 		if(!lying) //force them on the ground
-			lying = pick(90, 270)
+			if(src.dir == 1)
+				lying = pick(90, 270)
+			if(src.dir == 2)
+				lying = pick(90, 270)
+			if(src.dir == 4)
+				lying = 90
+			if(src.dir == 8)
+				lying = 270
```

Comment:
```suggestion
			switch(dir)
				if(NORTH, SOUTH)
					lying = pick(90, 270)
				if(EAST)
					lying = 90
				else //West
					lying = 270
```

Reply:
@Ghommie wouldn't it make more sense for North/South to be the else condition?